Resources/public/js/alloyeditor/plugins/yui3.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+/* global CKEDITOR */
+YUI.add('ez-alloyeditor-plugin-yui3', function (Y) {
+    "use strict";
+
+    if (CKEDITOR.plugins.get('yui3')) {
+        return;
+    }
+
+    function cleanUpIds(editor) {
+        Array.prototype.forEach.call(
+            editor.element.$.querySelectorAll('[id]'),
+            function (element) {
+                element.removeAttribute('id');
```

Comment:
that's a good remark. For now, there's no way to set an id on a link or an embed (I'm not even sure the PM wants that back). That said, I could have checked for the `yui_3_18` prefix but the same problem can also occur if the user paste some code where a strong (or any inline style element) has an id. So for now, IMHO, I think this is enough and if we add the id features on links and/or embed then, we'll have to improve the code.